setup.py

```diff
@@ -76,12 +50,54 @@ def get_requirements(requirements_file):
         include_package_data=True,
         zip_safe=False,
         scripts=["bin/modoboa-admin.py"],
-        install_requires=INSTALL_REQUIRES,
+        install_requires=[
```

Comment:
Why not move these to the [setup.cfg](https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files) file ?